README.md

```diff
@@ -1146,6 +1165,59 @@ terragrunt = {
 
 The common.tfvars located in the terraform root folder will be included by all applications, whatever their relative location to the root.
 
+#### get_terraform_commands_that_need_vars
+
+`get_terraform_commands_that_need_vars()`
+
+Returns the list of terraform commands that accept -var and -var-file parameters. This function is used when defining [extra_arguments](#keep-your-cli-flags-dry).
+
+```
+terragrunt = {
+  terraform = {
+    ...
+
+    extra_arguments "common_var" {
+      commands  = ["${get_terraform_commands_that_need_vars()}"]
+      arguments = ["-var-file=${get_aws_account_id()}.tfvars"]
+    }
+  }
+}
+```
+
+#### get_terraform_commands_that_need_locking
+
+`get_terraform_commands_that_need_locking()`
+
+Returns the list of terraform commands that accept -lock-timeout parameter. This function is used when defining [extra_arguments](#keep-your-cli-flags-dry).
+
+```hcl
+terragrunt = {
+  terraform {
+    # Force Terraform to keep trying to acquire a lock for up to 20 minutes if someone else already has the lock
+    extra_arguments "retry_lock" {
+      commands  = ["${get_terraform_commands_that_need_locking()}"]
+      arguments = ["-lock-timeout=20m"]
+    }
+  }
+}
+```
+
+_Note: Functions that return a list of values must be used in a single declaration like:_
+
+```hcl
+commands = ["${get_terraform_commands_that_need_vars()}"]
+
+# which result in:
+commands = ["apply", "console", "destroy", "import", "plan", "push", "refresh"]
+
+# They shall not be used to do string composition like:
```

Comment:
I changed it, but in fact, this is not resulting in an error. It is just that the result would be useless. That explain why I used shall not instead of cannot.